module-apps/application-settings/windows/display-keypad/DisplayLightWindow.cpp

```diff
@@ -118,6 +118,7 @@ namespace gui
         };
 
         auto spinner = std::make_unique<gui::SpinBoxOptionSettings>(
+            utils::translate("app_settings_display_light_brightness"),
             utils::translate("app_settings_display_light_brightness"),
```

Comment:
Should it be duplicated?

Reply:
As we discussed offline - yes ;) After my changes the `SpinBoxOptionSettings` constructor takes two text parameters - first one displayed when the option is not focused, second one when focused. In this case in code there's no need to show different texts, so I just passed the same translation twice.